src/utils/trusts.ts

```diff
@@ -1,14 +1,14 @@
-import { ApplicationData } from "model";
-import { BeneficialOwnerIndividual } from "model/beneficial.owner.individual.model";
-import { BeneficialOwnerOther } from "model/beneficial.owner.other.model";
-import { BeneficialOwnerItem } from "model/trust.model";
+import { ApplicationData } from "../model";
+import { BeneficialOwnerIndividual, BeneficialOwnerIndividualKey } from "../model/beneficial.owner.individual.model";
+import { BeneficialOwnerOther, BeneficialOwnerOtherKey } from "../model/beneficial.owner.other.model";
+import { BeneficialOwnerItem, TrustBeneficialOwner } from "../model/trust.model";
 
 // Checks whether any beneficial owners have trust data
-export const checkEntityHasTrusts = (appData: ApplicationData): boolean => {
+const checkEntityHasTrusts = (appData: ApplicationData): boolean => {
   if (appData) {
     const allBenficialOwnersToCheck: (BeneficialOwnerIndividual[] | BeneficialOwnerOther[] | undefined)[] = [
       appData.beneficial_owners_individual,
-      appData.beneficial_owners_corporate
+      appData.beneficial_owners_corporate,
```

Comment:
Why the extra comma at the end? Is this right? Sorry, my node knowledge not great!

Reply:
Thanks for explaining. Makes sense to me.